cluster/gce/util.sh

```diff
@@ -31,19 +31,16 @@ else
 fi
 
 if [[ "${OS_DISTRIBUTION}" == "gci" ]]; then
-  # If the master or node image is not set, we use the latest GCI dev image.
-  # Otherwise, we respect whatever set by the user.
+  # If the master image is not set, we use the latest GCI dev image.
+  # Otherwise, we respect whatever is set by the user.
   gci_images=( $(gcloud compute images list --project google-containers \
       --regexp='gci-dev.*' --format='value(name)') )
-  if [[ -z "${MASTER_IMAGE:-}" ]]; then
-    MASTER_IMAGE="${gci_images[0]}"
-    MASTER_IMAGE_PROJECT="google-containers"
-  fi
-  if [[ -z "${NODE_IMAGE:-}" ]]; then
-    NODE_IMAGE="${gci_images[0]}"
-    NODE_IMAGE_PROJECT="google-containers"
-  fi
-
+  MASTER_IMAGE=${KUBE_GCE_MASTER_IMAGE:-"${gci_images[0]}"}
+  MASTER_IMAGE_PROJECT=${KUBE_GCE_MASTER_PROJECT:-google-containers}
+  # The default node image when using GCI is still the Debian based ContainerVM
+  # until GCI gets validated for node usage.
+  NODE_IMAGE=${KUBE_GCE_NODE_IMAGE:-container-v1-3-v20160517}
```

Comment:
> A better way: leave the default ContainerVM image version defined in config-test.sh and config-default.sh, and use that variable here. 

Can you elaborate more? Did you mean keep `config-default.sh` and `config-test.sh` unchanged? That wouldn't work because before we switched to GCI as the default master image, there was that contract that node image would default to the master image. The sole purpose of this change is to bring that contract back for backward compatibility. See #26291.

> it looks not good to me that we put ContainerVM image info in a gci if block

I actually think it makes sense because this is the situation we are currently in: by default, we start a cluster with GCI on master and containervm on nodes.